CONTRIBUTING.md

```diff
@@ -99,6 +100,34 @@ Our different types of reviews:
   3. **Only comments.** You must address all the comments and need another review until you merge.
   4. **Request changes.** Only use if something critical is in the PR that absolutely must be addressed. We usually use `h` comments for that. When someone requests changes, the same person must approve the changes to allow merging. Use this sparingly.
 
+## Commit, Issue & PR guidelines
+
+### Commits
+
+For commit messages, we use the format:
+
+```
+<type>(<scope>): <subject> (<github-id>)
+````
+
+For example: `feat(core): Set custom transaction source for event processors (#5722)`.
```

Comment:
l: Should we point out that this format only strictly has to apply to the squashed commit that we merge into `master`? 

Asking because what I'm usually doing is that in the first commit of my branch, I use the `<type>(<scope>): <subject>` format but without any github id. For follow-up commits I usually don't stick strictly to the convention. Not saying that this is the best way to go but IMO, since we squash and get the chance to rewrite the message before merging, this is the only message that actually matters if that makes sense...

Reply:
Yeah, good point, I also thought about this a bit.

Maybe add a part like:

> The Github-ID can be left out until the PR is merged

Does that make sense?